### DISCUSSION TITLE
Changelogs on GUI

### DISCUSSION THREAD

**Discussion by erdnuesse** [AdguardTeam/AdGuardHome#5192] [Ideas]
Hi, 
based on the latest release frequency I find myself more often than not looking for a changelog of the pushed releases. 
For beta releases, I wouldn't mind, but on other free and open software I use, I have no trouble finding release notes for the current stable release. 
I would really like a Changelog entry at the bottom of the page that opens when I click on the build number, right next to the check for update button.
Feedback and/or considerations are appreciated.

**Comment by ppfeufer** [comment]
You can subscribe to releases on GitHub and you'll get it mailed to you with every release.

![image](https://user-images.githubusercontent.com/2989985/203994200-ac7e283a-c601-4af8-8cae-bed744f29db6.png)


![image](https://user-images.githubusercontent.com/2989985/203994359-2fad6fba-6850-4284-a1c1-cca680bb1e34.png)

**Comment by erdnuesse** [comment]
Hi, thanks for the idea; 
However, based on quasi-standards regarding almost every other app in existence, at least from my personal experience, it would be an improvement for user experience in general, if I have a link to click on the same popup, where I can actually click on it, rather than receiving an e-mail, which will arrive at any point that I need to look for behind all other emails; 

When someone focuses on their updates, there it should be located; not make a user jump through unecessary hoops. 
Even If the Link would redirect to the same source. 

Regards
